### DISCUSSION TITLE
dataset unit testing conformance auxiliary functions.

### DISCUSSION THREAD

**Discussion by mu578** [Reference-LAPACK/lapack#838] [General]
Hello,

is someone have some standalone unit-test input/output dataset (error rate, expectation...) for auxiliary functions such as \*lasq\* family any help appreciated.

Thank you.

**Comment by oamarques** [comment]
We do have a (Fortran) standalone test for the *lasq* family. We are mostly interested in accuracy and speed (in comparison with QR, for example). Is that what you mean by "error rate, expectation?"

**Comment by mu578** [comment]
Hello, thank you, ye, I am looking for a pre-computed set of diags of the bidiags matrices and their  expected outputs; for instance, in q2 there is a comment about single and ieee (I suspect that a 3 epsilons machine precision would do it, instead of mostly assuming that radix is always 2) so, having already checked data would help to compare and verify.

**Comment by mu578** [comment]
If you have fortran test routines; I am interested in; however, nothing available here; + it took me two days to enter *lasq* "code", what a nightmare :). Maybe having Jacobi iterations and unrolled steps in shape of routines would be better; instead of having something organized by dimension.
